test/k8sT/manifests/log-gatherer.yaml

```diff
@@ -30,8 +30,8 @@ spec:
         - "10000"
         command:
         - sleep
-        image: docker.io/cilium/log-gatherer:v1.0
-        imagePullPolicy: Always
+        image: docker.io/cilium/log-gatherer:v1.1
```

Comment:
This change is not part of the original commit. @nebril or @nbusseneau is it ok to bump the version or should we keep it to v1.0?

Reply:
If the CI passes it is safe to bump the version.